app/views/homepage/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Classroom Dashboard</title>
+  </head>
+  <body>
+    <div align="center">
+      <% if logged_in? %>
+        <h1><%= current_user["name"] %></h1>
+        <h4><%= current_user["email"] %></h4>
+        <%= link_to "View All Courses", user_courses_path(current_user["id"]), :class => "btn btn-lg" %>
+        <br>
+        <%= link_to "Log Out", logout_path, :class => "btn btn-danger" %>
+        
+      <% else %>
+        <%= link_to "Login", login_path, :class => "btn btn-lg" %>
+      <% end %>
+      <div class="container">
+        <% if flash.any? %>
+          <% flash.each do |type, message| %>
+            <div class="alert alert-<%= type %>"><%= message %></div>
+          <% end %>
+        <% end %>
+        <%= yield %>
+      </div>
+    </div>
+  </body>
+</html
```

Comment:
In some documents, like this one, end tags are aligned with the open tags, but in other documents the end tags are less indented by one space. We should probably be more consistent.